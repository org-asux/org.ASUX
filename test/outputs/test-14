MyRootELEMENT:
  MyCompositeELEMENT-0:
  - ZoneName: eu-north-1a
    ZoneId: eun1-az1
    State: available
    RegionName: eu-north-1
    Messages: [
      ]
  - ZoneName: eu-north-1b
    ZoneId: eun1-az2
    State: available
    RegionName: eu-north-1
    Messages: [
      ]
  - ZoneName: eu-north-1c
    ZoneId: eun1-az3
    State: available
    RegionName: eu-north-1
    Messages: [
      ]
  MyCompositeELEMENT-1:
  - ZoneName: ap-south-1a
    ZoneId: aps1-az1
    State: available
    RegionName: ap-south-1
    Messages: [
      ]
  - ZoneName: ap-south-1b
    ZoneId: aps1-az3
    State: available
    RegionName: ap-south-1
    Messages: [
      ]
  - ZoneName: ap-south-1c
    ZoneId: aps1-az2
    State: available
    RegionName: ap-south-1
    Messages: [
      ]
  MyCompositeELEMENT-2:
  - ZoneName: eu-west-3a
    ZoneId: euw3-az1
    State: available
    RegionName: eu-west-3
    Messages: [
      ]
  - ZoneName: eu-west-3b
    ZoneId: euw3-az2
    State: available
    RegionName: eu-west-3
    Messages: [
      ]
  - ZoneName: eu-west-3c
    ZoneId: euw3-az3
    State: available
    RegionName: eu-west-3
    Messages: [
      ]
  MyCompositeELEMENT-3:
  - ZoneName: eu-west-2a
    ZoneId: euw2-az2
    State: available
    RegionName: eu-west-2
    Messages: [
      ]
  - ZoneName: eu-west-2b
    ZoneId: euw2-az3
    State: available
    RegionName: eu-west-2
    Messages: [
      ]
  - ZoneName: eu-west-2c
    ZoneId: euw2-az1
    State: available
    RegionName: eu-west-2
    Messages: [
      ]
  MyCompositeELEMENT-4:
  - ZoneName: eu-west-1a
    ZoneId: euw1-az3
    State: available
    RegionName: eu-west-1
    Messages: [
      ]
  - ZoneName: eu-west-1b
    ZoneId: euw1-az1
    State: available
    RegionName: eu-west-1
    Messages: [
      ]
  - ZoneName: eu-west-1c
    ZoneId: euw1-az2
    State: available
    RegionName: eu-west-1
    Messages: [
      ]
  MyCompositeELEMENT-5:
  - ZoneName: ap-northeast-2a
    ZoneId: apne2-az1
    State: available
    RegionName: ap-northeast-2
    Messages: [
      ]
  - ZoneName: ap-northeast-2b
    ZoneId: apne2-az2
    State: available
    RegionName: ap-northeast-2
    Messages: [
      ]
  - ZoneName: ap-northeast-2c
    ZoneId: apne2-az3
    State: available
    RegionName: ap-northeast-2
    Messages: [
      ]
  MyCompositeELEMENT-6:
  - ZoneName: ap-northeast-1a
    ZoneId: apne1-az4
    State: available
    RegionName: ap-northeast-1
    Messages: [
      ]
  - ZoneName: ap-northeast-1c
    ZoneId: apne1-az1
    State: available
    RegionName: ap-northeast-1
    Messages: [
      ]
  - ZoneName: ap-northeast-1d
    ZoneId: apne1-az2
    State: available
    RegionName: ap-northeast-1
    Messages: [
      ]
  MyCompositeELEMENT-7:
  - ZoneName: sa-east-1a
    ZoneId: sae1-az1
    State: available
    RegionName: sa-east-1
    Messages: [
      ]
  - ZoneName: sa-east-1c
    ZoneId: sae1-az3
    State: available
    RegionName: sa-east-1
    Messages: [
      ]
  MyCompositeELEMENT-8:
  - ZoneName: ca-central-1a
    ZoneId: cac1-az1
    State: available
    RegionName: ca-central-1
    Messages: [
      ]
  - ZoneName: ca-central-1b
    ZoneId: cac1-az2
    State: available
    RegionName: ca-central-1
    Messages: [
      ]
  MyCompositeELEMENT-9:
  - ZoneName: ap-southeast-1a
    ZoneId: apse1-az1
    State: available
    RegionName: ap-southeast-1
    Messages: [
      ]
  - ZoneName: ap-southeast-1b
    ZoneId: apse1-az2
    State: available
    RegionName: ap-southeast-1
    Messages: [
      ]
  - ZoneName: ap-southeast-1c
    ZoneId: apse1-az3
    State: available
    RegionName: ap-southeast-1
    Messages: [
      ]
  MyCompositeELEMENT-10:
  - ZoneName: ap-southeast-2a
    ZoneId: apse2-az1
    State: available
    RegionName: ap-southeast-2
    Messages: [
      ]
  - ZoneName: ap-southeast-2b
    ZoneId: apse2-az3
    State: available
    RegionName: ap-southeast-2
    Messages: [
      ]
  - ZoneName: ap-southeast-2c
    ZoneId: apse2-az2
    State: available
    RegionName: ap-southeast-2
    Messages: [
      ]
  MyCompositeELEMENT-11:
  - ZoneName: eu-central-1a
    ZoneId: euc1-az2
    State: available
    RegionName: eu-central-1
    Messages: [
      ]
  - ZoneName: eu-central-1b
    ZoneId: euc1-az3
    State: available
    RegionName: eu-central-1
    Messages: [
      ]
  - ZoneName: eu-central-1c
    ZoneId: euc1-az1
    State: available
    RegionName: eu-central-1
    Messages: [
      ]
  MyCompositeELEMENT-12:
  - ZoneName: us-east-1a
    ZoneId: use1-az6
    State: available
    RegionName: us-east-1
    Messages: [
      ]
  - ZoneName: us-east-1b
    ZoneId: use1-az1
    State: available
    RegionName: us-east-1
    Messages: [
      ]
  - ZoneName: us-east-1c
    ZoneId: use1-az2
    State: available
    RegionName: us-east-1
    Messages: [
      ]
  - ZoneName: us-east-1d
    ZoneId: use1-az4
    State: available
    RegionName: us-east-1
    Messages: [
      ]
  - ZoneName: us-east-1e
    ZoneId: use1-az3
    State: available
    RegionName: us-east-1
    Messages: [
      ]
  - ZoneName: us-east-1f
    ZoneId: use1-az5
    State: available
    RegionName: us-east-1
    Messages: [
      ]
  MyCompositeELEMENT-13:
  - ZoneName: us-east-2a
    ZoneId: use2-az1
    State: available
    RegionName: us-east-2
    Messages: [
      ]
  - ZoneName: us-east-2b
    ZoneId: use2-az2
    State: available
    RegionName: us-east-2
    Messages: [
      ]
  - ZoneName: us-east-2c
    ZoneId: use2-az3
    State: available
    RegionName: us-east-2
    Messages: [
      ]
  MyCompositeELEMENT-14:
  - ZoneName: us-west-1a
    ZoneId: usw1-az3
    State: available
    RegionName: us-west-1
    Messages: [
      ]
  - ZoneName: us-west-1b
    ZoneId: usw1-az1
    State: available
    RegionName: us-west-1
    Messages: [
      ]
  MyCompositeELEMENT-15:
  - ZoneName: us-west-2a
    ZoneId: usw2-az2
    State: available
    RegionName: us-west-2
    Messages: [
      ]
  - ZoneName: us-west-2b
    ZoneId: usw2-az1
    State: available
    RegionName: us-west-2
    Messages: [
      ]
  - ZoneName: us-west-2c
    ZoneId: usw2-az3
    State: available
    RegionName: us-west-2
    Messages: [
      ]
  - ZoneName: us-west-2d
    ZoneId: usw2-az4
    State: available
    RegionName: us-west-2
    Messages: [
      ]
