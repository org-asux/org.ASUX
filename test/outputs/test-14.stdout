swagger: "2.0"
info:
  contact:
    email: sarma@ussmoso.com
    url: http://www.apache.org/licenses/LICENSE-2.0.html
basePath: /playground
paths:
  /pet:
    put:
      consumes:
      - application/json
      - - petId
        - integer
      responses:
        ${ASUX::Key202}:
          description: ${ASUX::Key101}
          schema:
            $ref: ${ASUX::Key202}

swagger: "2.0"
info:
  contact:
    email: sarma@ussmoso.com
    url: http://www.apache.org/licenses/LICENSE-2.0.html
basePath: /playground
paths:
  /pet:
    put:
      consumes:
      - - petId
        - integer
      - application/json
      responses:
        ${ASUX::Key202}:
          description: ${ASUX::Key101}
          schema:
            $ref: ${ASUX::Key202}

swagger: "2.0"
info:
  contact:
    email: sarma@ussmoso.com
    url: http://www.apache.org/licenses/LICENSE-2.0.html
basePath: /playground
paths:
  /pet:
    put:
      consumes:
      - application/json
      responses:
        ${ASUX::Key202}:
          description: ${ASUX::Key101}
          schema:
            $ref: ${ASUX::Key202}
      '1':
      - petId
      - integer

swagger: "2.0"
info:
  contact:
    email: sarma@ussmoso.com
    url: http://www.apache.org/licenses/LICENSE-2.0.html
basePath: /playground
paths:
  /pet:
    put:
      consumes:
      - - petId
        - integer
      responses:
        ${ASUX::Key202}:
          description: ${ASUX::Key101}
          schema:
            $ref: ${ASUX::Key202}

'MyRootELEMENT': "dlfkjhasdlfhasdfadsfafsdfasdfasdfasdfasdf"

MyRootELEMENT:
  "subElem": "dlfkjhasdlfhasdfadsfafsdfasdfasdfasdfasdf"

"AdditionalRootElement":
  "State": "available"
  "Messages":
  - "A"
  - "B"
  - "C"
  "RegionName": "eu-north-1"
  "ZoneName": "eu-north-1c"
  "ZoneId": "eun1-az3"

	Echo: yaml --insert MyRootELEMENT.subElem.leafElem '{State: "available", Messages: ["A","B","C"], RegionName: "eu-north-1", ZoneName: "eu-north-1c", ZoneId: "eun1-az3"}' --no-quote
MyRootELEMENT:
  subElem:
    leafElem:
      State: available
      Messages:
      - A
      - B
      - C
      RegionName: eu-north-1
      ZoneName: eu-north-1c
      ZoneId: eun1-az3

	Echo: yaml --insert / !AdditionalRootElement --no-quote
"MyRootELEMENT": ""
"AdditionalRootElement":
  "State": "available"
  "Messages":
  - "A"
  - "B"
  - "C"
  "RegionName": "eu-north-1"
  "ZoneName": "eu-north-1c"
  "ZoneId": "eun1-az3"

	Echo: yaml --insert / !AdditionalRootElement --double-quote
MyRootELEMENT: "\t"
"AdditionalRootElement":
  "State": "available"
  "Messages":
  - "A"
  - "B"
  - "C"
  "RegionName": "eu-north-1"
  "ZoneName": "eu-north-1c"
  "ZoneId": "eun1-az3"

	Echo: yaml --insert / !AdditionalRootElement --no-quote
- petId
- integer
- "AdditionalRootElement":
    "State": "available"
    "Messages":
    - "A"
    - "B"
    - "C"
    "RegionName": "eu-north-1"
    "ZoneName": "eu-north-1c"
    "ZoneId": "eun1-az3"

	Echo (As-Is): yaml --insert Resources @${ASUX::AWSCFN.HOME}/AWSCFN.templates-pieces/PublicSubnet.yaml -i test/inputs/Snippet-Resources.yaml
	Echo (Macros-substituted): yaml --insert Resources @/Users/Sarma/Documents/Development/src/org.ASUX/AWS/CFN/AWSCFN.templates-pieces/PublicSubnet.yaml -i test/inputs/Snippet-Resources.yaml
eu-north-1 ap-south-1 eu-west-3 eu-west-2 eu-west-1 ap-northeast-2 ap-northeast-1 sa-east-1 ca-central-1 ap-southeast-1 ap-southeast-2 eu-central-1 us-east-1 us-east-2 us-west-1 us-west-2 
